lsp-release.sh

```diff
@@ -85,100 +85,54 @@ TEMPDIR=$(mktemp -d -t lsp-release.XXXX)
 echo "Using ${TEMPDIR}"
 
 function tmp_clone() {
-	cd $TEMPDIR
-	git clone git@github.com:influxdata/$1.git &> /dev/null
-	echo "$(pwd)/$1"
+    cd $TEMPDIR
+    git clone git@github.com:influxdata/$1.git &> /dev/null
+    echo "$(pwd)/$1"
 }
 
 LSP_DIR=$(tmp_clone flux-lsp)
 UI_DIR=$(tmp_clone ui)
 
-function tag_lsp_release() {
-	v=v`version $LSP_DIR/Cargo.toml`
-	git tag -a -s $v -m "Release $v"
-	git push origin master $v
-	flux_version=v$(grep -m 1 'flux = ' $LSP_DIR/Cargo.toml | version)
-	hub release create $v -m "Release $v
+cd $LSP_DIR
+LSP_VERSION=v$(version Cargo.toml)
 
-	- Upgrade to [Flux $flux_version](https://github.com/influxdata/flux/releases/tag/$flux_version)" -e
-}
+echo "Cutting release for Flux LSP v$LSP_VERSION"
+git tag -a -s $LSP_VERSION -m "Release $LSP_VERSION"
+git push origin master $LSP_VERSION
 
-cd $LSP_DIR
-lsp_version=$(version Cargo.toml)
+LSP_PREVIOUS_VERSION=`git describe --abbrev=0 ${LSP_VERSION}^`
+COMMITS=`git log --pretty=oneline ${LSP_PREVIOUS_VERSION}...${LSP_VERSION} | awk '{$1="-"; print }'`
```

Comment:
This was my biggest hang-up about the previous release script. Glad this is resolved.